docs/core/Graph.md

```diff
@@ -48,7 +48,7 @@ Graph is a set of graphs that share a common set of base visualization structure
 | showNoDataText         | boolean  | true                                           | Toggle to show no data text                                                                                                                                                             |
 | showShapes             | boolean  | true                                           | Toggle to show shapes in the line graph                                                                                                                                                 |
 | showVGrid              | boolean  | true                                           | Toggle to show vertical grid                                                                                                                                                            |
-| ticksCount             | number   | 3                                              | Number of ticks to show on the vertical axes, excluding the upper and lower limits, if not provided, the number of ticks will be calculated automatically depending on the Y & Y2 range |
+| ticksCount             | number   | 3-7 depending on the input range               | Number of ticks to show on the vertical axes, excluding the upper and lower limits, if not provided, the number of ticks will be calculated automatically depending on the Y & Y2 range |
```

Comment:
So based on the documentation, the consumer can only provider ticksCount between 3 and 7. Does it include 3 and 7 values as well?

Reply:
Oh, I mean the default values are from 3-7. Otherwise the acceptable range is form 0-19. I'll probably rephrase it to make it clearer.